doc/source/whatsnew/v0.18.1.txt

```diff
@@ -76,6 +76,25 @@ API changes
 
 
 
+- ``read_csv`` now raises an ``EmptyDataError``, a subclass of ``ValueError``, in response to empty columns or header (:issue:`12506`)
+
+.. ipython:: python
+
+   from pandas import read_csv
```

Comment:
you don't actually need the import at all. We are already importing this in the docs (and there is a note already). These docs should be really simple / straightforwar. Look at the v0.18.0 and see how the sub-sections are done.